bftengine/src/bcstatetransfer/BCStateTran.hpp

```diff
@@ -114,15 +114,15 @@ class BCStateTran : public IStateTransfer {
   void saveReservedPage(uint32_t reservedPageId, uint32_t copyLength, const char* inReservedPage) override;
   void zeroReservedPage(uint32_t reservedPageId) override;
 
-  // Handoff API- handle timeouts and messages from an external context
-  void onTimer() override { handoff_->push(std::bind(&BCStateTran::onTimerImp, this)); };
+  // Incoming Events (Handoff) - ST main thread is a consumer of timeouts and messages arriving from an external context
+  void onTimer() override { IncomingEventsQ_->push(std::bind(&BCStateTran::onTimerImp, this)); };
   using LocalTimePoint = time_point<steady_clock>;
   static constexpr auto UNDEFINED_LOCAL_TIME_POINT = LocalTimePoint::max();
   void handleStateTransferMessage(char* msg, uint32_t msgLen, uint16_t senderId) override {
-    handoff_->push(std::bind(
+    IncomingEventsQ_->push(std::bind(
         &BCStateTran::handleStateTransferMessageImp, this, msg, msgLen, senderId, std::chrono::steady_clock::now()));
   };
-
+  std::unique_ptr<concord::util::Handoff> IncomingEventsQ_;
```

Comment:
Nit: IncomingEventsQ_ -> incomingEventsQ_

Reply:
That's strange, I was fixing this issue few times and then forgot to fix it before pushing.
thanks for noticing, saved me another PR.